modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockStorage.scala

```diff
@@ -271,6 +321,21 @@ object BlockStorage {
     val errorMessage: String =
       s"Block with hash=${hash.show} failed to transition state to Accepted! Encountered state: ${encountered.show}."
   }
+
+  case class BlockWaitingToPostponedError(hash: ProofsHash, encountered: Option[StoredBlock])(implicit s: Show[StoredBlock])
```

Comment:
this also sounds weird, like with missing verb or sth

Reply:
Changed.